reference/docs-conceptual/dsc/configurations/flow-control-in-configurations.md

```diff
@@ -6,14 +6,23 @@ title:  Conditional statements and loops in Configurations
 
 # Conditional statements and loops in Configurations
 
-You can make your [Configurations](configurations.md) more dynamic using PowerShell flow-control keywords. This article will show you how you can use conditional statements, and loops to make your Configurations more dynamic. Combining conditional and loops with [parameters](add-parameters-to-a-configuration.md) and [Configuration Data](configData.md) allows you more flexibility and control when compiling your Configurations.
+You can make your [Configurations](configurations.md) more dynamic using PowerShell flow-control
+keywords. This article shows you how you can use conditional statements and loops to make your
+`Configurations` more dynamic. Combining conditional statements and loops with [parameters](add-parameters-to-a-configuration.md)
```

Comment:
Since the keyword is not plural we should change this to:

This article shows you how you can use conditional statements and loops to make your `Configuration` more dynamic.